cypress/support/step_definitions/validations.js

```diff
@@ -336,26 +336,3 @@ Given(
     });
   }
 );
-
```

Comment:
Why are we removing this?

Reply:
Why do we need to move this to configHelper? There is nothing proprietary in it.